blockchain/chain/src/store/errors.rs

```diff
@@ -72,3 +76,9 @@ impl From<CidErr> for Error {
         Error::Cid(e.to_string())
     }
 }
+
+impl From<AmtErr> for Error {
+    fn from(_e: AmtErr) -> Error {
+        Error::Amt
+    }
+}
```

Comment:
I think this is probably a bad example of forwarding an error, since this adds no value and instead drops the error message from the AMT, which will be very unhelpful